library/gpg/test/gpg_test.rb

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env rspec
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require_relative "test_helper"
+require "tempfile"
+
+Yast.import "GPG"
+
+describe Yast::GPG do
+  let(:file_content) do
+    "-----BEGIN PGP MESSAGE-----
```

Comment:
Um, why not having it as a plain data file? That would avoid that save/remove `around` block...

Like we have https://github.com/yast/yast-yast2/tree/master/library/packages/test/y2packager/zypp/test_repo/repodata or https://github.com/yast/yast-yast2/tree/master/library/packages/test/data/rpm

Reply:
well, I consider it small enough, but moving is ok for me